src/bootstrap.jl

```diff
@@ -243,6 +245,24 @@ function tidyβ(bsamp::MixedModelBootstrap{T}) where {T}
     result
 end
 
+function coeftable(bsamp::MixedModelBootstrap{T}) where {T}
```

Comment:
if we're going to name this `coeftable`, should we follow the `StatsBase.coeftable` API and return a `CoefTable` object? See e.g. the [method for `LinearMixedModel`](https://github.com/JuliaStats/MixedModels.jl/blob/9b7a748080c0f7904f00adde17a716db43caa26a/src/linearmixedmodel.jl#L197-L211)?

Reply:
Are you thinking of a vector of `coeftable` of `CoefTable` objects?  A single `CoefTable` of `nsamp * ncoef` rows might be unwieldy to work with.  Currently the returned object is a `Vector{NamedTuple}`, which is a row table, and can be converted to other tabular forms such as a DataFrame.  My recollection is that a `CoefTable` is not as easy to work with.  It was primarily created for its `show` method.

Reply:
d'oh. I wasn't thinking clearly / hadn't read carefully enough. Returning a single `CoefTable` of that size would be unwieldy. Likewise for returning a `Vector{CoefTable}`. I don't know what the right course is here. The current variant is nice in terms of memory and dependencies, but does drift awy from the `StatsBase.coeftable` API expectation.